src/app/components/BCP47Tag.svelte

```diff
@@ -1,11 +1,20 @@
 <script lang="ts">
   import bcp47Parser from "bcp-47";
+  import worker from "../spawn-worker";
 
   const TAG_INPUT_ID = "tag-input";
 
   let bcp47Tag = "";
+  let validTag = false;
+  let displayEnterBCP47Tag = true;
```

Comment:
I added this variable to control the displayed message, since I think we should display 'Enter BCP 47 tag' when the input is empty.

Reply:
Excellent!